VirtoCommerce.Platform.Web/Scripts/app/exportImport/blades/export-main.js

```diff
@@ -10,6 +10,8 @@
         exportImportResourse.getNewExportManifest(function (data) {
             $scope.exportRequest.exportManifest = data;
             blade.isLoading = false;
+        }, function (error) {
```

Comment:
Could you check why the common platform error handling doesn't work for this actions?
https://github.com/VirtoCommerce/vc-platform/blob/master/VirtoCommerce.Platform.Web/Scripts/app/app.js#L59

Reply:
It worked, in fact, so reverted redundant handlers. Also changed error handler for FileUploader to produce error with non empty details.